packages/central-server/src/tests/apiV2/changes/getChanges.test.js

```diff
@@ -74,6 +75,10 @@ describe('GET /changes/count', async () => {
 
     // Add some more questions
     await oneSecondSleep();
```

Comment:
Can we get rid of oneSecondSleep now? :)

Reply:
I don't think we can 😞 The ChangeHandler won't schedule an execution until it receives a notification from the database that a change has occurred. If for some reason it takes the database longer to send a notification than it does for node to get to the next line, then calling `waitForAllChangeHandlers()` will complete instantly as no ChangeHandlers have been scheduled.

There's multiple levels of async stuff going on here 😅 It might just work though... I'm not really sure... but I don't think it'd be reliable...